packages/react-redux/src/slices/cartoSlice.js

```diff
@@ -254,7 +254,7 @@ export const selectIsViewportFeaturesReadyForSource = (state, id) =>
 
 const debouncedSetViewPort = debounce((dispatch, setViewPort) => {
   dispatch(setViewPort());
-}, 200);
+}, 25);
```

Comment:
what happens if we remove the debounce? I've seen that kepler doesn't use a debounce for this and we have a debounce in the viewport features calculation

Reply:
I tried removing the debounce but this action is called many many many times while zooming and panning, I dont think we should update redux state so fastly. That's why I think it's convenience to have at least a very small debounce. Also, in viewport features the behavior was weird. I felt the app strange. I don't remember the details, but with this debounce, it worked perfectly.